### ISSUE TITLE
No completion results are shown in macro definitions

### ISSUE THREAD

**Issue by sean-mcmanus** [microsoft/vscode-cpptools#9078]
@sean-mcmanus Thanks for your consideration.
I think it is the same bug.
simple macro definitions will not have any autocomplete while writing them
```c
#define INVERTER_BASE        (0x80)
#define HAL_RX               (0x01+INVERTER_BASE)  // INVERTER_BASE will not be autocompleted even if declared as a variable
```
function renaming
```
#include "task.h"
#define os_add_task   xTaskCreate //same issue
```

I meant by "pages of macro constants"  the register map file for microcontroller registers. ex: CMSIS.
when making modifications or making your own it is very important to have IntelliSense.

many thanks

_Originally posted by @AshrafMasoud5 in https://github.com/microsoft/vscode-cpptools/issues/5616#issuecomment-1076135522_

**Comment by sean-mcmanus** [comment]
This is similar to https://github.com/microsoft/vscode-cpptools/issues/4662 , except we were using that issue to track completion in the name part of the macro and not the value/definition (not sure yet if they're the same case).

**Comment by sean-mcmanus** [comment]
@AshrafMasoud5 This should be fixed with 1.10.2 (pre-release).
